communication/slack-config/usergroups.yaml

```diff
@@ -56,8 +56,9 @@ usergroups:
       - joelsmith
       - lukehinds
       - micahhausler
+      - ritazh
+      - SaranBalaji90
```

Comment:
Their Slack user IDs needs to be added to https://github.com/kubernetes/community/blob/master/communication/slack-config/users.yaml as well.

Reply:
Whoops, this was also a github ID, not a slack ID. Fixed.